howtos/add-new-repository.md

```diff
@@ -0,0 +1,13 @@
+
+# Add a new repository to Eiffel Community
+
+Want to extend the Eiffel community with another project?
```

Comment:
I think we need to clarify the distinction between "project" and "repository" because this and following sections use "project" and repository interchangeably which may not be always the case. The reason for this is that one project may consist of more than one repository - this is already the case today and an example to this is RemREM.

So the question I want to ask is whether we should use a language that uses "project" as the thing we talk about and document instead of "repository"? 

A related discussion is happening under project lifecycle and it makes more sense to apply the lifecycle to project as a whole rather than individual repositories. [0]

If this makes sense, the document could be updated and instead worded in a way that people contact the community to request creation of a new project which comes with one associated repository by default. If they require more than one repository, that could be requested later.
[0] https://github.com/eiffel-community/eiffel-repository-template/issues/9

Reply:
Indeed it might be a good idea to not mix the two words. For this document I think repository is the word to use, since it's a repository (and not a project) inside the eiffel-community organization which is being created. 

Regarding you comment about REMReM, yes the repositories are connected but they are not part of a GitHub project.  From what I can see we don't have any projects within the eiffel-community organization: https://github.com/orgs/eiffel-community/projects 
Perhaps this question how to handle such requests can be discussed/decided on by the (Bootstrap) Technical Committe.

For now this guide will refer to creating one or multiple repositories. I'm not really familiar with the setting up of projects within a GitHub organization. I'll leave it to somebody else to dig into how to set up GitHub projects and update this guide later 😄 .  

Regarding your last comment about the lifecycle discussion, I assume this signal would then be placed inside each repository? - and if we indeed make it a proper template like suggested in https://github.com/eiffel-community/eiffel-repository-template/issues/7  this would be included in any new repositories.

Reply:
I agree that we need to be clear on what we mean here, and I believe @e-pettersson-ericsson is right in that we want to refer to creating 'repositories' in this howto.
@fdegir, when you say that 'RemREM' is an example of a project, what do you mean? I can't see that it is a 'GitHub project'.

Reply:
@e-backmark-ericsson the word "project" i use(d) in this context is "something to group related repositories that makes up a project" rather than "project in github".

I mean we need a way to group related repositories together and instead switch how we work with projects rather than individual repositories. I agree that it is not related with this PR and it is a more general conversation to take.